README.md

```diff
@@ -49,7 +53,11 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town.residents.delete_at(1)
+residents = {:Belle}
+castle = {:"Robby Benson", :"Belle"}
+castle.merge(residents)
+
```

Comment:
```rb
town[:residents].delete_at(1)
town[:castle][:guests].push("Belle")
```

Don't get strings and symbols mixed up! None of the names here should have colons in front of them; they're all better as strings.

Also, curly brackets are only used for hashes; square brackets are for arrays.